src/coredata.cs

```diff
@@ -2520,11 +2571,31 @@ interface NSBatchInsertRequest {
 		[Export ("batchInsertRequestWithEntityName:objects:")]
 		NSBatchInsertRequest BatchInsertRequest (string entityName, NSDictionary<NSString, NSObject>[] dictionaries);
 
+		[DesignatedInitializer]
 		[Export ("initWithEntityName:objects:")]
 		IntPtr Constructor (string entityName, NSDictionary<NSString, NSObject>[] dictionaries);
 
+		[DesignatedInitializer]
 		[Export ("initWithEntity:objects:")]
 		IntPtr Constructor (NSEntityDescription entity, NSDictionary<NSString, NSObject>[] dictionaries);
+
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[NullAllowed, Export ("dictionaryHandler", ArgumentSemantic.Copy)]
+		NSBatchInsertRequestDictionaryHandler DictionaryHandler { get; set; }
+
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[NullAllowed, Export ("managedObjectHandler", ArgumentSemantic.Copy)]
+		NSBatchInsertRequestManagedObjectHandler ManagedObjectHandler { get; set; }
+
+		[Watch (7,0), TV (14,0), Mac (11,0), iOS (14,0)]
+		[Static]
+		[Export ("batchInsertRequestWithEntityName:dictionaryHandler:")]
+		NSBatchInsertRequest BatchInsertRequest (string entityName, NSBatchInsertRequestDictionaryHandler handler);
```

Comment:
no verb, `Insert` is not a verb here since it return a type of that name,
-> `CreateBatchInsertRequest`